.github/workflows/itests.yml

```diff
@@ -41,21 +41,21 @@ jobs:
           install-version: '7.0.x'
     env:
       NUPKG_OUTDIR: bin/Release/nugets
-      GOVER: 1.19
+      GOVER: 1.20.3
       GOOS: linux
       GOARCH: amd64
       GOPROXY: https://proxy.golang.org
       DAPR_CLI_VER: 1.9.1
       DAPR_RUNTIME_VER: 1.9.5
       DAPR_INSTALL_URL: https://raw.githubusercontent.com/dapr/cli/3dacfb672d55f1436c249057aaebbe597e1066f3/install/install.sh
       DAPR_CLI_REF: ''
-      DAPR_REF: '82e097134cdc7494c5c3be71bf80f7904d6db9ea'
+      DAPR_REF: 'dbeea96710d38d08dcc5aaf6807914a7d697bed4'
     steps:
       - name: Set up Dapr CLI
         run: wget -q ${{ env.DAPR_INSTALL_URL }} -O - | /bin/bash -s ${{ env.DAPR_CLI_VER }}
       - name: Set up Go ${{ env.GOVER }}
         if: env.DAPR_REF != '' || env.DAPR_CLI_REF != ''
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v3
```

Comment:
Can we do it like this instead? Then we don't need to keep updating the go version. You'll have to move this step to in between pulling and building dapr.

https://github.com/dapr/test-infra/blob/master/.github/workflows/dapr-longhaul-nightly.yml#L58